stream.go

```diff
@@ -110,8 +111,7 @@ type ClientStream interface {
 	// It should not be called until after Header or RecvMsg has returned. Once
 	// called, subsequent client-side retries are disabled.
 	Context() context.Context
-	// SendMsg is generally called by generated code. On success, SendMsg returns
```

Comment:
Only one addition is required here

// SendMsg is generally called by generated code. On success, SendMsg returns. On success, SendMsg returns nil 
// for unary and server-streaming RPCs. On error, SendMsg aborts the stream. If the error was generated by the 
// client, the status is returned directly; otherwise, io.EOF is returned and the status of the stream may be
// discovered using RecvMsg. However, if `StreamDesc.ClientStreams` was set to false, then nil error is returned 
// unconditionally.